python/src/phylanx.cpp

```diff
@@ -321,6 +325,26 @@ PYBIND11_MODULE(_phylanx, m)
             &phylanx::bindings::pickle_helper<phylanx::ast::expression>,
             &phylanx::bindings::unpickle_helper<phylanx::ast::expression>));
 
+    // list of phylanx::ast::operations
+    pybind11::class_<std::list<phylanx::ast::operation>>(ast,"operation_list","A list of operations")
+        .def(pybind11::init<>())
+        .def("pop_back",&std::list<phylanx::ast::operation>::clear)
```

Comment:
This should probably call `std::list<>::pop_back()`, not `std::list<>::clear()`.

Reply:
I fixed this.